code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -259,27 +259,18 @@
 	var/old_heat_capacity = air.heat_capacity()
 	var/temperature = air.temperature
 
-	var/burned_fuel
-	var/fire_scale
-	if(cached_gases[/datum/gas/oxygen][MOLES] < cached_gases[/datum/gas/hydrogen][MOLES] || MINIMUM_HYDROGEN_OXYBURN_ENERGY > (temperature * old_heat_capacity))
-		burned_fuel = cached_gases[/datum/gas/oxygen][MOLES] / HYDROGEN_BURN_OXY_FACTOR // const must be at least one
-		fire_scale = 1
+	var/burned_fuel = min(cached_gases[/datum/gas/hydrogen][MOLES] / FIRE_HYDROGEN_BURN_RATE_DELTA, cached_gases[/datum/gas/oxygen][MOLES] / (FIRE_HYDROGEN_BURN_RATE_DELTA * HYDROGEN_OXYGEN_FULLBURN), cached_gases[/datum/gas/hydrogen][MOLES], cached_gases[/datum/gas/oxygen][MOLES] * INVERSE(0.5))
```

Comment:
The burn rate here is insane, at least if I'm reading it right. 1/2 hydrogen in one tick seems insane, especially in bomb making. the o2 is a bit better and I get that it's acting as the weight on hydrogen's ankle here but it's still a bit worrying.
Why exactly is it so high?

Reply:
I want it to be unique by having its extremely fast reaction rate to release energy quickly and explosively, rather than just being plasma but releasing more energy. Its energy release has been decreased, making a fixed amount of a hydrogen and oxygen mix release a similar (burns to around 70,000K water vapour) amount of energy as a plasma oxygen (over 90,000K 25% water vapour 75% co2) mix. I haven't done extensive testing, but have managed to get the anomaly refinery to report an explosion size of 103 (I assume devastation range) with tier 2 cooled t2/o2 and 4,999K plasma TTV.